Source/items.cpp

```diff
@@ -1741,76 +1741,103 @@ void DrawUniqueInfoWindow(const Surface &out)
-				AddPanelString(_("Right-click to read"));
-			} else {
-				AddPanelString(_("Right-click to read, then"));
-				AddPanelString(_("left-click to target"));
-			}
-		} else {
-			if (TargetsMonster(item._iSpell)) {
-				AddPanelString(_("Select from spell book, then"));
-				AddPanelString(_("cast spell to read"));
-			} else if (!invflag) {
-				AddPanelString(_("Open inventory to use"));
-			} else {
-				AddPanelString(_("Activate to read"));
-			}
-		}
+	if (item._iMiscId == IMISC_MAPOFDOOM) {
+		AddPanelString(_("Right-click to view"));
+
+	} else if (isOil) {
+		PrintItemOil(item._iMiscId);
+		AddPanelString(_("Right-click to use"));
+	} else if (isCastOnTarget) {
+		AddPanelString(_("Right-click to read, then"));
+		AddPanelString(_("left-click to target"));
+	} else if (IsAnyOf(item._iMiscId, IMISC_BOOK, IMISC_NOTE, IMISC_SCROLL)) {
+		AddPanelString(_("Right-click to read"));
 	}
-	if ((item._iMiscId >= IMISC_USEFIRST && item._iMiscId <= IMISC_USELAST)
-	    || (item._iMiscId > IMISC_OILFIRST && item._iMiscId < IMISC_OILLAST)
-	    || (item._iMiscId > IMISC_RUNEFIRST && item._iMiscId < IMISC_RUNELAST)) {
+}
+
+void printItemMiscVirtualGamepad(const Item &item, const bool isOil)
+{
+	if (item._iMiscId == IMISC_MAPOFDOOM) {
+		AddPanelString(_("Activate to view"));
+	} else if (isOil) {
 		PrintItemOil(item._iMiscId);
-		if (ControlMode == ControlTypes::KeyboardAndMouse) {
-			AddPanelString(_("Right-click to use"));
+		if (!invflag) {
+			AddPanelString(_("Open inventory to use"));
+
 		} else {
-			if (!invflag) {
-				AddPanelString(_("Open inventory to use"));
-			} else {
-				AddPanelString(_("Activate to use"));
-			}
+			AddPanelString(_("Activate to use"));
 		}
+	} else if (item._iMiscId == IMISC_SCROLL) {
+		AddPanelString(_("Select from spell book, then"));
+		AddPanelString(_("cast to read"));
+	} else {
+		AddPanelString(_("Activate to read"));
 	}
-	if (IsAnyOf(item._iMiscId, IMISC_BOOK, IMISC_NOTE)) {
-		if (ControlMode == ControlTypes::KeyboardAndMouse) {
-			AddPanelString(_("Right-click to read"));
-		} else {
-			AddPanelString(_("Activate to read"));
-		}
+}
+
+#ifndef USE_SDL1
+void printItemMiscGamepad(const Item &item, bool isOil, bool isCastOnTarget)
+{
+	std::string y_icon = "Y";
+	std::string x_icon = "X";
+
+	if (GamepadType == GamepadLayout::Xbox) {
+		y_icon = "Y";
+		x_icon = "X";
+	} else if (GamepadType == GamepadLayout::Playstation) {
+		y_icon = "Triangle";
+		x_icon = "Square";
+	} else if (GamepadType == GamepadLayout::Nintendo) {
+		y_icon = "Y";
+		x_icon = "X";
 	}
+
 	if (item._iMiscId == IMISC_MAPOFDOOM) {
-		if (ControlMode == ControlTypes::KeyboardAndMouse) {
-			AddPanelString(_("Right-click to view"));
+		AddPanelString(_(y_icon + " to view"));
+	} else if (isOil) {
+		PrintItemOil(item._iMiscId);
+		if (!invflag) {
+			AddPanelString(_("Open inventory to use"));
 		} else {
-			AddPanelString(_("Activate to view"));
+			AddPanelString(_(y_icon + " to use"));

```

Comment:
`_()` is a translation function, you need to feed it string that can be picked up directly from the source or it will not match any string in the translation file. So it has to be either the ful finalized string, or a paramters inserted using `fmt::format()`.